toplevel/coqinit.ml

```diff
@@ -38,9 +38,7 @@ let load_rcfile doc sid =
 	try
 	  let warn x = Feedback.msg_warning (str x) in
 	  let inferedrc = List.find CUnix.file_readable_p [
-	    Envars.xdg_config_home warn / rcdefaultname^"."^Coq_config.version;
 	    Envars.xdg_config_home warn / rcdefaultname;
-	    Envars.home ~warn / "."^rcdefaultname^"."^Coq_config.version;
```

Comment:
Is this a change that eliminates a feature?  Previously could I have version-specific `~/.coqrc` files, and now I can't?

Reply:
That "feature" seems very fishy to me, that being said, we can restore that indeed.

Reply:
The problem is that "version" as understood by configure and "version" as understood by the tag system are pretty different to me.